osu.Game.Rulesets.Osu/Edit/OsuSelectionHandler.cs

```diff
@@ -363,6 +363,7 @@ private void mergeSelection()
                 return;
 
             ChangeHandler?.BeginChange();
+            EditorBeatmap.BeginChange();
```

Comment:
It shouldn't be required to call both of these. `EditorBeatmap.BeginChange()` implies the `ChangeHandler` call automatically.

https://github.com/ppy/osu/blob/3e246caf05ade38bffdc436fdc4e306d1c01dd2e/osu.Game/Screens/Edit/EditorChangeHandler.cs#L58-L60

Same goes for `EndChange()`, obviously.

Reply:
ah I didn't know that

Reply:
Is there a reason the change from `ChangeHandler` to `EditorBeatmap` was preferred here? It has very nuanced different behaviour which shouldn't really matter too much, but curious if that was a consideration here.

(I really don't think both of these should exist, and we should only expose this in one way..)

Reply:
Okay, so it actually fixes the whole flow in this case. Welp.